filters/accesslog/log.go

```diff
@@ -0,0 +1,50 @@
+/*
+Package accesslog provides a request filter that gives ability to override global access log setting.
+*/
+package accesslog
+
+import (
+	"github.com/zalando/skipper/filters"
+)
+
+const (
+	// AccessLogName is the filter name seen by the user
+	AccessLogName = "accessLog"
+
+	// AccessLogEnabledKey is the key used in the state bag to pass the access log state to the proxy.
+	AccessLogEnabledKey = "statebag:access_log:proxy:enabled"
+)
+
+type accessLog struct {
+	status bool
+}
+
+// NewAccessLog creates a filter spec for overriding the state of the access log. (By default global setting is used.)
+//
+// 	accessLog("false")
+//
+// EXPERIMENTAL: this filter is experimental, and the name and the arguments can change until marked as stable.
```

Comment:
I would go as far as to not even mark it as experimental. @vecmezoni @szuecs , do you foresee any reason to change the behavior/name/args in the future?

Reply:
I don't see it no.

Reply:
I would think of a more generic approach when you can even provide log format here. Maybe it can be done with the separate filter.